parsec/backend/memory/block.py

```diff
@@ -69,6 +71,11 @@ def _check_realm_access(
             realm = self._realm_component._get_realm(organization_id, realm_id)
```

Comment:
should pass now here

Reply:
4bdf0e2407b0936afe3dfa7ab9b30161cddea744